docs/api/apiv3/paths/days_non_working.yml

```diff
@@ -0,0 +1,48 @@
+# /api/v3/days/non_working
```

Comment:
There are a lot of duplicate routes between the different titles:
<img width="1203" alt="image" src="https://user-images.githubusercontent.com/83396/168081945-40bec3ca-d3d8-4aef-bb19-3aa2fff60f58.png">

<img width="1233" alt="image" src="https://user-images.githubusercontent.com/83396/168082138-07512029-7c18-4555-9c32-eec2b8ba04de.png">
<img width="1235" alt="image" src="https://user-images.githubusercontent.com/83396/168082183-cfa0d9b6-4c56-49d0-9ba3-d01bd5d113cc.png">
<img width="1239" alt="image" src="https://user-images.githubusercontent.com/83396/168082219-fc4c44d9-bd1d-4e5a-a92c-fa64418fd431.png">

Reply:
I think that's me not knowing how the OpenAPI `tags` should be used. I'll have to fix it. And also add a description to these tags.

Reply:
Fixed in 54ee9fc31b

There is only one "Work Schedule" section now with a brief description of the intended behavior.